group_fs_go1.16_test.go

```diff
@@ -0,0 +1,106 @@
+//go:build go1.16
+// +build go1.16
+
+package echo
+
+import (
+	"github.com/stretchr/testify/assert"
+	"io/fs"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+)
+
+func TestGroup_FileFS(t *testing.T) {
+	var testCases = []struct {
+		name             string
+		whenPath         string
+		whenFile         string
+		whenFS           fs.FS
+		givenURL         string
+		expectCode       int
+		expectStartsWith []byte
+	}{
+		{
+			name:             "ok",
+			whenPath:         "/walle",
+			whenFS:           os.DirFS("_fixture/images"),
+			whenFile:         "walle.png",
+			givenURL:         "/assets/walle",
+			expectCode:       http.StatusOK,
```

Comment:
This seems to be mixed up. OK should be returned when givenURL matches whenFile is suppose.
The next test should be OK at least from the params

Reply:
nope, `givenURL` is URL that you request and it should match `"assets/" + whenPath` to serve always the same file `whenFile`. Methods `g.File` and `g.FileFS` are used to make "aliases" to files.

this is how test set up the route
```go
			g := e.Group("/assets")
			g.FileFS(tc.whenPath, tc.whenFile, tc.whenFS)
```

Reply:
Ah ok, got confused as we are usually using "walle.png" for OK tests and "walle" for failed tests in the other test files.
But of course tests are ok.